torch/_dynamo/utils.py

```diff
@@ -906,6 +908,20 @@ def add_compilation_metrics_to_chromium(c: Dict[str, Any]) -> None:
         dynamo_config=c["dynamo_config"],
     )
 
+def scrubbed_inductor_config_for_logging() -> Dict[str, Any]:
```

Comment:
nit: can you prefix this function with underscore to (hopefully) communicate that it's "private" to this file?